README.md

```diff
@@ -1,10 +1,45 @@
 # Theia
 
-Theia contains Network Flow Visibility functions which are exacted from the
-[Antrea main repo](https://github.com/antrea-io/antrea). While flow exporter and
-flow aggregator are kept in the Antrea repo, other flow visibility modules, such
-as ClickHouse and Grafana related ones, will be moved to this new repo.
-
-During the code migration period, we will still keep related functions available
-and stable in the Antrea repo, and the functions in this new repo won't be ready
-until we announce the migration is completed.
+Theia is a network observability and analytics platform for Kubernetes. It is
```

Comment:
@salv-orlando : I believe you can write a better version. But could we start from the simple introduction?

Reply:
How about "provide fine-grained visibility into communications among Pods and Services in a Kubernetes cluster"? I feel people might think connections as TCP connections.

Reply:
just propose "visibility on workload among Pods, Service and external network" as alternative.